<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepBreath</class>
 <widget class="QMainWindow" name="DeepBreath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeepBreath</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_camera_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dimension_2d_radio_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dimension_button_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="dimension_3d_radio_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dimension_button_group</string>
    </attribute>
   </widget>
   <widget class="QPlainTextEdit" name="dimension_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Dimesion:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mode_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mode_combo_box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Distances</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Locations</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fourier</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Volume</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No Graph</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="locations_text">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Locations:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="left_loc_checkbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="right_loc_checkbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mid1_loc_checkbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mid1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mid2_loc_checkbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mid2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mid3_loc_checkbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mid3</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="num_markers_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>#Markers:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="num_markers_combo_box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="is_stickers_checkbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stickers</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="y_color_radio_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">color_button_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="g_color_radio_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">color_button_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="b_color_radio_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">color_button_group</string>
    </attribute>
   </widget>
   <widget class="QPlainTextEdit" name="distances_text">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Distances:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="left_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mid1_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>mid1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="right_text">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mid2_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>mid2</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mid3_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>mid3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="left_mid1_checkbox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="right_mid1_checkbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="left_right_checkbox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="mid1_mid2_checkbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="mid2_mid3_checkbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="right_mid2_checkbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="left_mid2_checkbox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="mid1_mid3_checkbox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="left_mid3_checkbox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="right_mid3_checkbox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="color_stream_widget" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="depth_stream_widget" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="volume_stream_widget" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graph_widget" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>350</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dimension_button_group"/>
  <buttongroup name="color_button_group"/>
 </buttongroups>
</ui>
